newsroom/default_settings.py

```diff
@@ -255,21 +250,32 @@
 }
 
 
+LANGUAGES = ['en', 'fi', 'cs', 'fr_CA']
+DEFAULT_LANGUAGE = 'en'
+
+CLIENT_LOCALE_FORMATS = {
+    "en": {  # defaults
+        "TIME_FORMAT": "HH:mm",
+        "DATE_FORMAT": "DD/MM/YYYY",
+        "COVERAGE_DATE_TIME_FORMAT": "HH:mm DD/MM",
+        "COVERAGE_DATE_FORMAT": "DD/MM",
+    },
+    "fr_CA": {  # example - you can overwrite any format above
+        "DATE_FORMAT": "DD/MM/YYYY",
+    }
+}
+
 # Client configuration
 CLIENT_CONFIG = {
-    'time_format': CLIENT_TIME_FORMAT,
-    'date_format': CLIENT_DATE_FORMAT,
-    'coverage_date_time_format': CLIENT_COVERAGE_DATE_TIME_FORMAT,
-    'coverage_date_format': CLIENT_COVERAGE_DATE_FORMAT,
+    'default_language': DEFAULT_LANGUAGE,
+    'locale_formats': CLIENT_LOCALE_FORMATS,
     'coverage_types': COVERAGE_TYPES,
     'display_abstract': DISPLAY_ABSTRACT,
     'list_animations': True,  # Enables or disables the animations for list item select boxes,
     'display_news_only': True,  # Displays news only switch in wire,
     'default_timezone': DEFAULT_TIMEZONE
 }
 
-LANGUAGES = ['en', 'fi', 'cs', 'fr_CA']
-DEFAULT_LANGUAGE = 'en'
```

Comment:
why removing these?

Reply:
I just moved them [below](https://github.com/superdesk/newsroom/pull/1033/files#diff-a9726395d27997f1b1342236c7e5539fR253-R254), so it's near the client config